ev3/legoev3_motor.c

```diff
@@ -57,12 +57,11 @@
 #include "../motors/ev3_motor.h"
 
 #define TACHO_MOTOR_POLL_NS	2000000	/* 2 msec */
+#define TACHO_MOTOR_POLL_MS	2000	/* 2 msec */
```

Comment:
Can one of these `POLL_?S` be removed? Or define one in terms of the other? Or if they are completely unrelated, change the names so that we know what the difference is.